volto_addon/{{ cookiecutter.addon_name }}/README.md

```diff
@@ -11,7 +11,11 @@
 
 <!-- List your awesome features here -->
 
-## Install
+## Installation
+
+### Into a @plone/generator-volto Volto based project
```

Comment:
I don't think the implementation detail is helpful for most users and will just confuse most of them. Let's stick to a Volto version, unless there is a very good reason to use an implementation detail.

```suggestion
### Volto 17 and earlier
```